docs/js/release-notes.mdx

```diff
@@ -46,21 +46,14 @@ For your convenience we [pre-generate type-safe clients libraries](https://www.n
 ### New functionality
 
 - **We released a regular update for pre-generated type-safe client libraries (also known as VDM) for the [latest RTC release 2008 of SAP S/4HANA Cloud](https://news.sap.com/2020/08/sap-s4hana-cloud-release-2008-stay-ahead-change/).**
-  - The update covers all the changes to existing services and introduces new ones. You'll find them in `com.sap.cloud.sdk.s4hana.datamodel.odata.services` and `com.sap.cloud.sdk.s4hana.datamodel.odatav4.services`. Below is the quick reference on difference and compatibility between 2005 and current 2008 S/4HANA Cloud releases:
-  - These deprecated OData **services are no longer allowed**:
-    - Removed deprecated `ChangeMasterService`, use the successor "[ChangeMasterServiceV2Service](https://api.sap.com/api/API_CHANGEMASTER_0002/resource)" instead.
-    - Removed deprecated `ProcessOrderConfirmationService`, use the successor "[ProcessOrderConfirmationV2Service](https://api.sap.com/api/API_PROC_ORDER_CONFIRMATION_2_SRV/resource)" instead.
-    - Removed deprecated `ProductionOrderConfirmationService`, use the successor "[ProductionOrderConfirmationV2Service](https://api.sap.com/api/API_PROD_ORDER_CONFIRMATION_2_SRV/resource)" instead.
-  - These OData service **methods have a changed argument order**:
-    - In service `SalesDocumentWithCreditBlocksService` the methods `rejectCreditBlock` and `releaseCreditBlock` are updated.
-    - In service `CustomerReturnsDeliveryV2Service` the methods `setPutawayQuantityWithBaseQuantity`, `putawayOneItemWithBaseQuantity` and `putawayOneItemWithSalesQuantity` are updated.
-  - These OData service **methods have been removed**:
-    - In service `ProductionOrderConfirmationV2Service` the method `createProdnOrdConfMatlDocItm` has been removed.
-    - In service `ProcessOrderConfirmationV2Service` the method `createProcOrdConfMatlDocItm` has been removed.
-    - In service `SupplierInvoiceIntegrationService` the methods `createSuplrInvcItemAcctAssgmt` and `createSuplrInvcItemPurOrdRef` have been removed.
-  - For reference on what OData services are included in current release, check out [SAP API Business Hub for SAP S/4HANA Cloud](https://api.sap.com/shell/discover/contentpackage/SAPS4HANACloud)
-
-### Compatibility notes
-
-- Despite releasing a `Beta` version of OData v4 type-safe client we didn't include pre-generated libraries for available S/4HANA OData v4 services in this release. We work to reach `General Availability` for OData v4 client soon and release a type-safe client for OData v4 services together with this milestone. We'll communicate it in the release notes.
-- We recommend updating to SAP Cloud SDK version 1.26.1 to benefit from the latest functionality of type-safe clients.
+  - The update covers all the changes to existing services and introduces new ones. You'll find them on [NPMJS](https://www.npmjs.com/search?q=%40sap%2Fcloud-sdk-vdm-*)
+
+## Version 1.21.0 - November 5, 2020
```

Comment:
The vdm release is not done, so the date might be changed.

Reply:
Version 1.21.0 ist the old version and was released 2 months ago. Don't have the exact date in mind.